### DISCUSSION TITLE
Issues with insufficientPermissions

### DISCUSSION THREAD

**Discussion by hkhumaini** [freekmurze/laravel-analytics#498] [Q&A]
{ "error": { "code": 403, "message": "User does not have sufficient permissions for this profile.", "errors": [ { "message": "User does not have sufficient permissions for this profile.", "domain": "global", "reason": "insufficientPermissions" } ] } }

Any idea how to fix this . I followed every step stated in readme but unable to output page views . For user the readme stated role Analyst is enough (thats what I did ) but i get this error messages . Appreciate if anyone can help

**Comment by ilhmhdyat** [comment]
same error

<img width="1364" alt="image" src="https://github.com/spatie/laravel-analytics/assets/41753591/bd91c445-5552-4a5e-8706-15b74fd90667">
can anyone help ?

**Reply by boonstoppel** [reply]
Same for me.

**Reply by EldenChris** [reply]
Same here...

**Reply by grycroft** [reply]
Same here.

**Reply by i1yaz** [reply]
Same here

**Comment by polearnik** [comment]
check package version . it should be 5 not 3. It was breaking changes like using property  id not view id. I spent about 2 days with this error.

**Reply by hkhumaini** [reply]
may I know which package should be 5. 
is it spatie/laravel-analytics? mine currently is 4.1

**Reply by polearnik** [reply]
https://github.com/spatie/laravel-analytics/releases show the latest version is 5.0.3
